### DISCUSSION TITLE
提议：修改用户未安装组件时的提示

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#3932] [功能建议 / Ideas]
![图片](https://user-images.githubusercontent.com/82081432/203472425-7f394d49-4ba3-4dcd-9916-100e1cdb61fd.png)
如图，可以将这个界面的提示改为：

空空如也哦 =￣ω￣=
你可以前往 设置>组件 来添加功能

感觉挺多人装了插件之后不知道怎么用，只会拿着类似上面的截图一顿喷

**Comment by Ethkuil** [comment]
这个挺好改的，我就顺手改掉了：`空空如也哦 =￣ω￣= 全新安装的脚本没有任何功能，你可以在设置面板 左下角 点击进入「组件」面板，再点击「在线」，从在线功能库任意安装自己想要的功能~`

---

话说确实好多情况我都想扔个「你tm不会看文档吗😅」

**Reply by Geeyun-JY3** [reply]
干脆直接用 https://github.com/the1812/Bilibili-Evolved#设置 那段吧，直接给说全说清楚了。

***

完整引用 README 已是我最后的耐性（
